saleor/graphql/shop/mutations.py

```diff
@@ -108,3 +108,69 @@ def mutate(cls, root, info, collection):
             return HomepageCollectionUpdate(errors=errors)
         site_settings.save(update_fields=['homepage_collection'])
         return HomepageCollectionUpdate(shop=Shop(), errors=errors)
+
+
+class AuthorizationKeyInput(graphene.InputObjectType):
+    key = graphene.String(description='Client authorization key (client ID).')
+    password = graphene.String(description='Client secret.')
+
+
+class AuthorizationKeyAdd(BaseMutation):
+    authorization_key = graphene.Field(
+        AuthorizationKey, description='Newly added authorization key.')
+
+    class Meta:
+        description = 'Adds an authorization key.'
+
+    class Arguments:
+        key_type = AuthorizationKeyType(
+            required=True, description='Type of authorization key to add.')
```

Comment:
I think we are missing `an` before the `authorization` (as in the description below)